pkgs/applications/misc/redis-desktop-manager/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config qmake ];
   buildInputs = [
     libssh2 qtbase qtdeclarative qtgraphicaleffects qtimageformats
-    qtquick1 qtquickcontrols qtsvg qttools qtcharts
+    qtquick1 qtquickcontrols qtquickcontrols2 qtsvg qttools qtcharts
```

Comment:
do we need qtquickcontrols and qtquickcontrols2?

Reply:
I'm pretty sure it does. it uses controls from both versions ([ImageButton](https://github.com/uglide/RedisDesktopManager/blob/0.9.1/src/qml/common/ImageButton.qml#L2) uses v2, [most other controls](https://github.com/uglide/RedisDesktopManager/blob/0.9.1/src/qml/common/AddressInput.qml#L3) uses v1). To double check, I'm trying to build without the `qtquickcontrols`, but I expect it to fail to work.

Reply:
Hmm, nvm , you appear to be right, just using qtquickcontrols2 appear to be enough.